dig/dig.go

```diff
@@ -157,7 +157,7 @@ func (g *graph) Reset() {
 	g.Lock()
 	defer g.Unlock()
 
-	defaultGraph.nodes = make(map[interface{}]object)
+	g.nodes = make(map[interface{}]object)
```

Comment:
I am just fixing them along the way. We have too many bugs to open a PR for each bug :)